src/gstreamer/KvsSinkStreamCallbackProvider.cpp

```diff
@@ -71,8 +71,12 @@ STATUS
 KvsSinkStreamCallbackProvider::streamClosedHandler(UINT64 custom_data,
                                                    STREAM_HANDLE stream_handle,
                                                    UPLOAD_HANDLE upload_handle) {
-    UNUSED_PARAM(custom_data);
-    LOG_DEBUG("Reported streamClosed callback for stream handle " << stream_handle << ". Upload handle " << upload_handle);
```

Comment:
Is the upload_handle irrelevant debug information?

Reply:
Yeah. It does not give us any useful details. I can add it back though. I wanted to ensure stream name was getting logged because that is descriptive